### PR TITLE
Increase test cover

### PR DESCRIPTION
**Pull request by marioidival**

### CODE REVIEW

File: adapters/postgres/postgres.go [hunk 1]
```diff
@@ -200,14 +196,7 @@ func CountByRequest(req *http.Request) (countQuery string) {
 
 // Query process queries
 func Query(SQL string, params ...interface{}) (jsonData []byte, err error) {
-	validQuery := chkInvalidIdentifier(SQL)
```
Comment (felipeweb) [inline adapters/postgres/postgres.go]: why you remove this this func prevent SQL injections should not be removed

File: controllers/tables.go [hunk 2]
```diff
@@ -22,13 +22,7 @@ func GetTables(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	order, err := postgres.OrderByRequest(r)
```
Comment (felipeweb) [inline controllers/tables.go]: Never ignore errors! Please check all ignored errors

Comment (marioidival) [inline controllers/tables.go]: Check `OrderByRequest` function... There is no way to generate errors.

File: adapters/postgres/postgres.go (hunk 1 above)
Comment (marioidival) [inline adapters/postgres/postgres.go]: [check here](https://github.com/nuveo/prest/pull/69/commits/a84dc741586b4f028d911d721df506bd40f25ff3)

File: controllers/tables.go (hunk 2 above)
Comment (felipeweb) [inline controllers/tables.go]: but is a good practice check all errors

File: adapters/postgres/postgres.go (hunk 1 above)
Comment (felipeweb) [inline adapters/postgres/postgres.go]: now i see, so you have to add the checker on order by if not has yet, sorry for that

File: controllers/tables.go (hunk 2 above)
Comment (marioidival) [inline controllers/tables.go]: Yes, I focgot to remove

**Comment by avelino** [comment]
Thank you @marioidival

**Pull request merged**